src/Mod/TechDraw/Gui/QGVPage.cpp

```diff
@@ -1244,4 +1253,49 @@ QColor QGVPage::getBackgroundColor()
     return fcColor.asValue<QColor>();
 }
 
+double QGVPage::getDevicePixelRatio() const {
+    for (Gui::MDIView *view : m_vpPage->getDocument()->getMDIViews()) {
+        if (view->isDerivedFrom(Gui::View3DInventor::getClassTypeId())) {
+            return static_cast<Gui::View3DInventor *>(view)->getViewer()->devicePixelRatio();
+        }
+    }
+
+    return 1.0;
+}
+
+QPixmap QGVPage::prepareCursorPixmap(const char *iconName, QPoint &hotspot) {
+    double cursorSize = 64.0;
```

Comment:
Please add comments for all numbers you choose. It is not directly clear why 64 us used here and not e.g. 60.